README.md

```diff
@@ -73,10 +72,12 @@ export class BaseService {
 | `updateField<T>`       | Updates fields in the document referred to by this DocumentReference in a fieldPath.                                           | `collection: string, id: string, fieldName: keyof T, value: any`                                                                  | `Promise`      |
 | `remove`               | Deletes the document referred to by this DocumentReference.                                                                    | `collection: string, id: string`                                                                                                  | `Promise`      |
 | `exists`               | True if the document exists.                                                                                                   | `collection: string, id: string`                                                                                                  | `Promise`      |
+| `getSizeCollection`               | Return size of collection.                                                                                                   | `collection: string, options?: Options<T>`                                                                                                  | `Promise<number>`      |
```

Comment:
Please adjust the description "Return size of collection." to "Return the size of the collection."